tests/backend_test_base.py

```diff
@@ -342,7 +344,8 @@ def run_test_case(self, func, feed_dict, input_names_with_port, output_names_wit
                                      const_node_values=const_node_values,
                                      initialized_tables=initialized_tables,
                                      **process_args)
-                g = optimizer.optimize_graph(g, catch_errors=False)
+                if optimize:
```

Comment:
True, I disabled the optimization because the final graph is different from what the converter generates and it was more difficult to debug. I let it because I thought other people might be interested into that option.